### ISSUE TITLE
Performance in practice: how to make a super fast site

### ISSUE THREAD

**Issue by AshKyd** [BrisJS/meetups#260]
There's a bunch of performance info out there. Can someone distill this down to a top 10 performance tips for a speedy site?

**Comment by drpitman76** [comment]
This is very old, if no-one has covered this yet (or recently) and people are interested I would be happy to do this one anytime...

**Comment by theharold9000** [comment]
Hey @drpitman76, picking back up on this - I think there'd be a fair bit of interest in there, are you still happy to present a talk on the topic?

**Comment by drpitman76** [comment]
Yes, I would - but I've presented twice in the last two meetups so I thought it would be good to take a rest for a few months before doing anymore...  I actually have an idea for this present as a lightning talk, but was going to wait a bit to suggest it.  My focus would be on things that can be done right in the client-side HTML,   which is often overlooked.

**Comment by mbourne** [comment]
I'm happy to do a session on this. Maybe August?

**Comment by iamkevinv** [comment]
@mbourne - always happy to receive a new entry to schedule. We are in need of talks.

**Comment by mbourne** [comment]
How about Part 2 of SVGs in August, then the Optimisation one in September?

**Comment by iamkevinv** [comment]
Lets have them both on the issues board and we can figure out the scheduling from there, but yes, I wouldn't mind your suggestion at all.
